db/migrate/20180302105225_add_two_factor_backupable_to_user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTwoFactorBackupableToUser < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :otp_backup_codes, :string, array: true
```

Comment:
Looks like it has to be a `text` as `string` is too short.

Reply:
cool, I'll do that